src/main/java/com/xceptance/neodymium/module/StatementBuilder.java

```diff
@@ -35,6 +36,22 @@
         {
             throw new RuntimeException(e);
         }
+        catch (IllegalArgumentException e)
+        {
+            throw new RuntimeException(e);
+        }
+        catch (InvocationTargetException e)
+        {
+            throw new RuntimeException(e);
+        }
+        catch (NoSuchMethodException e)
+        {
+            throw new RuntimeException(e);
+        }
+        catch (SecurityException e)
+        {
+            throw new RuntimeException(e);
+        }
```

Comment:
Why don't you catch all exceptions in one multi-catch clause?

Reply:
done